usecases/auth/authentication/apikey/db_users.go

```diff
@@ -106,13 +112,30 @@ func NewDBUser(path string) (*DBUser, error) {
 	if snapshot.Data.UserKeyRevoked == nil {
 		snapshot.Data.UserKeyRevoked = make(map[string]struct{})
 	}
-
-	return &DBUser{
+	dbUsers := &DBUser{
 		path:          fullpath,
 		lock:          &sync.RWMutex{},
 		data:          snapshot.Data,
 		memoryOnyData: memoryOnlyData{WeakKeyStorageById: make(map[string][sha256.Size]byte)},
-	}, nil
+	}
+
+	// we save every change to file after a request is done, EXCEPT the lastUsedAt time as we do not want to write to a
+	// file with every request.
+	// This information is not terribly important (besides WCD UX), so it does not matter much if we very rarely loose
+	// some information here. This info will also be written on shutdown so the only loss of information occurs with
+	// OOM or similar.
+	enterrors.GoWrapper(func() {
+		ticker := time.NewTicker(1 * time.Minute)
+		for range ticker.C {
+			func() {
+				dbUsers.lock.RLock()
+				defer dbUsers.lock.RUnlock()
+				_ = dbUsers.storeToFile()
```

Comment:
this method returns an error in case of failed write, we don't need to care about errors here, yes?

Reply:
I added a logger in case of an error